api/envoy/extensions/common/tap/v3/common.proto

```diff
@@ -42,4 +42,5 @@ message AdminConfig {
   // Opaque configuration ID. When requests are made to the admin handler, the passed opaque ID is
   // matched to the configured filter opaque ID to determine which filter to configure.
   string config_id = 1 [(validate.rules).string = {min_len: 1}];
+  uint64 max_concurrent_streams = 2 [(validate.rules).uint64 = {gt: 0}];
```

Comment:
Other `max_concurrent_streams` values in the APIs use `google.protobuf.UInt32Value`, e.g. https://github.com/envoyproxy/envoy/blob/d82b5a02befbff3b2119b309e865f7b44d207907/api/envoy/config/core/v3/protocol.proto#L63.

So I suggest we use the same.

Also, can we add a comment for what this new field means?  And is there a default value if it's not specified?

Also, I think `gte: 1` makes it clearer than `gt: 0` (it took me a couple seconds to realize the value must be >= 1), but that's more of a personal preference.  The couple other instances of `max_concurrent_streams` in proto definitions also use `gte: 1`

Reply:
That make sense, I'll add some comments for this new field.